Homework_2.0.py

```diff
@@ -17,38 +17,16 @@ def __init__(self, name, position):
         self.position = self.check_ascii(position)
 
     def check_ascii(self, param):
-        if type(param) == str:
-            if param.isascii():
-                return param
-            return Exception("String is not ascii!")
+        if isinstance(param, str) and param.isascii():
+            return param
+        return Exception("String is not ascii!")
```

Comment:
The Exception could be not a string so the message is incorrect, I would change it to `f'Expected ascii string, received: {param}'`